build/Makefile

```diff
@@ -354,6 +355,24 @@ else
 endif
 	echo "Finishing e2e controller failure test!"
 
+test-e2e-ha-extensions: FEATURE_GATES ?= $(ALPHA_FEATURE_GATES)
```

Comment:
Non-blocking: I would like us to look at reducing redundancy here since we're repeating ourselves a lot. It's not clear to me that the original split couldn't be done a different way - it might be nice to try folding all of this back into the main e2e package. But certainly this is fine for this PR.